src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = { ...state };
```

Comment:
Also, you can try to use const in this case (optional), for objects and arrays better way not use reassigns)
Thanks)